src/main/java/bio/terra/cda/app/util/QueryTranslator.java

```diff
@@ -19,6 +20,15 @@ public static String sql(String table, Query query) {
     return new SqlGenerator(table, query).generate();
   }
 
+  /** Split the TABLE.ColumnName into component parts */
+  public static Map<String, String> parseTableName(String qualifiedTable) {
+    int dotPos = qualifiedTable.lastIndexOf('.');
+    String table = qualifiedTable.substring(0, dotPos);
+    String column = qualifiedTable.substring(dotPos + 1);
+
+    return Map.of("columnName", qualifiedTable.substring(dotPos + 1), "tableName", table);
```

Comment:
can use `column` here
```suggestion
    return Map.of("columnName", column, "tableName", table);
```

Reply:
refactored